server/app.js

```diff
@@ -60,7 +60,9 @@ io.on('connection', (socket) => {
         if (result.matchFound && result.userData.isLoggedIn) {
             socket.emit("alert text", "You've already logged in. Please sign out of other sessions before trying again.");
         } else if (result.matchFound && result.banned) {
-            socket.emit("banned");
+            let updatedUser = UserUtils.updateUserLoginInfo(result.userData, true, socket.id);
```

Comment:
What happens when the user wants to leave the banned page? Does their login status update to "false"?

Reply:
yes

Reply:
to clarify if they refresh or go back then it logs them out, but the navbar functionally is different throughout the website. on some pages its not responsive, and on others it navigates out but does/doesn't log people out.

Reply:
That'll be fixed in my PR hopefully